### DISCUSSION TITLE
why is an automation assigned to workstations executed by servers also ?

### DISCUSSION THREAD

**Discussion by Runningman-work** [amidaware/tacticalrmm#1785] [Q&A]
Tactical RMMv0.17.5

We have an active automation policy with a run-once-task wich we wanted to be executed for workstations only. so I assigned the policy to only workstations for the site. but it is effectively executed for our servers also. what is my mistake ?

regards,
Stefan

**Comment by silversword411** [comment]
Post pictures showing the automation policies on the agent,agent type, and automation policy so we can replicate.

**Reply by Runningman-work** [reply]
Agent type is Windows 10, server is Linux. picture of automation is attached.
![checkmk1](https://github.com/amidaware/tacticalrmm/assets/139329588/170a116e-0aa1-4ec3-856f-3790533192dd)

On the agent the executed policy shows like this:
![checkmk2](https://github.com/amidaware/tacticalrmm/assets/139329588/6793ac16-4667-4472-baae-1d47ac19527b)

**Reply by dinger1986** [reply]
Please show policies applied and machines in the correct tabs.

Are servers in servers? Workstations jn workstations?

**Reply by Runningman-work** [reply]
ok my bad, I did not use an automation to execute the script, I used a buld script like in this window:
![checkmk4](https://github.com/amidaware/tacticalrmm/assets/139329588/36c30c3a-d655-4c68-b3e4-888d18f15b31)

I did test it with an automation, that worked correctly.

**Reply by dinger1986** [reply]
ah ok, so a bulk script got it to run on windows and linux? Did you set your script to windows when you created it?

![image](https://github.com/amidaware/tacticalrmm/assets/7244447/67bd9ba7-1b1e-47a0-a4e5-eeb34a07951d)

**Reply by Runningman-work** [reply]
yes Script was set to Windows

**Reply by dinger1986** [reply]
so I can replicate what was the settings you used on Bulk Script?
